cmd/main.go

```diff
@@ -68,6 +84,17 @@ func init() {
 	rootCmd.Flags().Int64Var(&maxVolumesPerNode, "maxvolumespernode", 0, "limit of volumes per node")
 	rootCmd.Flags().StringVar(&shareRelistInterval, "share-relist-interval", "",
 		"the time between controller relist on the share resource expressed with golang time.Duration syntax(default=10m")
+	rootCmd.Flags().BoolVar(&readOnly, "readonly", false, "does not watch for resource updates")
+	rootCmd.Flags().StringSliceVar(&ignoredNamespaces, "ignorenamespace", []string{}, "namespaces ignored by the controller")
```

Comment:
I wonder if in addition to the command line arg, we document and employ a specific config map in the openshift-cluster-csi-drivers namespace that our driver uses for analogous config for his stuff, we associated keys in the config map for each of these params

we would also establish an order of precedence between command line arg and config map

Reply:
continue to have our default, non "system" OCP namespaces for now, though over time, we will probably have to adjust this as the rotating cert scenario withing OCP starts leveraging our stuff

Reply:
> I wonder if in addition to the command line arg, we document and employ a specific config map in the openshift-cluster-csi-drivers namespace that our driver uses for analogous config for his stuff, we associated keys in the config map for each of these params

I would be a good addition to this project, I think we should pursue this as a next step. I have a few consideration on what it entails and how to approach using a `ConfigMap` for this role, should we discuss during office hours?

Reply:
@adambkaplan are you good with deferring the use of ConfigMap to configure the CSI driver to a work item outside of @otaviof 's current story here?

Reply:
Yes - let's split configuration via ConfigMap/file mount a separate work item.

Reply:
New issue in JIRA: https://issues.redhat.com/browse/BUILD-340

Reply:
perfect thanks @adambkaplan 

feel free to resolve this or consider it processed in however fashion you prefer @otaviof - thanks